### DISCUSSION TITLE
The output is not correct

### DISCUSSION THREAD

**Discussion by suiyixiaobai** [seung-lab/connected-components-3d#92] [Q&A]
Hi, I input a larger block of voxels and split the result into images. But I found something wrong with the order at the end. For example, the size of the voxel block is (224, 512, 512). But the 69th picture has become the 159th picture. I don't understand why this is.

**Comment by william-silversmith** [comment]
Hi suiyixiaobai, can you provide some sample code to reproduce the issue? It's pretty difficult for me to understand what's going wrong otherwise.
